src/data/mainnet/tokens-info.json

```diff
@@ -406,7 +417,8 @@
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/mcREAL.png",
     "name": "Moola interest bearing CREAL",
     "symbol": "mCREAL",
-    "isSupercharged": true
+    "isSupercharged": true,
+    "pegTo": "0xe8537a3d056da446677b9e9d6c5db704eaab4787"
```

Comment:
I don't know why this value was not set.

Reply:
...Can you move this change to a separate PR?